src/main/kotlin/week1/step3/Car.kt

```diff
@@ -0,0 +1,19 @@
+package week1.step3
+
+class Car {
+    var forwardCount = 1
+        private set
+
+    fun updateForwardCount() {
+        if (availableForward()) {
+            forwardCount++
+        }
+    }
+
+    private fun availableForward(): Boolean = WINNER_NUMBER_RANGE.random() > WINNER_NUMBER_CUT_LINE
+
+    companion object {
+        private val WINNER_NUMBER_RANGE = 0..9
+        private const val WINNER_NUMBER_CUT_LINE = 3
```

Comment:
협업하는 팀원이 쉽게 읽을 수 있는 코드로 변경해 보세요.

```suggestion
        private const val WINNER_NUMBER_CUT_LINE = 4
```

Reply:
수정하였습니다.
[210c1e83f4e85770be7ea3555deab3566b8b5fc0]